src/lib/api/tests/addons.test.ts

```diff
@@ -171,7 +171,7 @@ describe("addon payloads", () => {
     const payload = addons.buildPayload(scraper, form, true);
 
     if (payload.errors) {
-      console.error(payload.errors);
+      console.warn(payload.errors);
```

Comment:
Not sure if this should change, as it's in a test. Do we want to be logging in tests at all?

Reply:
I think I had that in so I could debug while working on validation. Don't think we need it now.